### DISCUSSION TITLE
How to return to default homebrew installation of php

### DISCUSSION THREAD

**Discussion by samhsabin** [shivammathur/homebrew-php#1850] [Q&A / Help] [answered]
This repository is great, and really useful for accessing previous version of PHP. I needed to use PHP 7 for a deprecated program, and now I want to return to the current version of PHP. Is there a way to revert the homebrew links to their original destination(like how it would normally be installed with `brew install php`)? I'm not super familiar with homebrew so sorry if this request is a bit simple.

**Comment by shivammathur** [comment, marked as answer]
You can install the latest php version and switch to it.

```zsh
brew install php
brew link --force --overwrite php
```

If you get an error like php is already linked, unlink it and link it again.

```zsh
brew unlink php
brew link --force --overwrite php
```

Restart your terminal, it should now have the latest php version.

If you no longer need the old PHP version like `php@7.0`, you can remove it
```zsh
brew remove php@7.0
```

**Reply by samhsabin** [reply]
Thanks! This is exactly what I needed
